media-libs/opencollada/opencollada-1.6.23.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/KhronosGroup/OpenCOLLADA.git"
```

Comment:
There is no reason to have this in the global scope for this ebuild. May I suggest a single if block for the live/non-live bits for this purpose, similar to the one in https://github.com/gentoo/gentoo/blob/master/dev-vcs/gitstats/gitstats-0_pre20151223.ebuild ?